### DISCUSSION TITLE
Exclude myself using Adblock Plus

### DISCUSSION THREAD

**Discussion by jx42** [plausible/analytics#560] [Anything Else]
I'd like to exclude myself from the analytics data of my website. I'm using Adblock Plus in Firefox but it doesn't block the plausible script. I didn't have any success with adding a custom filter as described here: https://plausible.io/docs/excluding/

Does anybody have any idea how ensure that Adblock Plus blocks the plausible script?

**Comment by metmarkosaric** [comment]
I can see that Adblock Plus allows you to add custom filters too. You can learn more [here about how they do it](https://adblockplus.org/filter-cheatsheet). Something like this will work (just tested it on Firefox myself):

||plausible.io^$script,domain=yourdomain.com

Or if you're using our custom domain feature, use this:

||yoursubdomain.yourdomain.com^$script,domain=yourdomain.com

**Reply by jx42** [reply]
Awesome! Works perfectly and thanks for the quick answer 👍 Maybe you can add this to the docs. Might be useful for others, too.

**Reply by metmarkosaric** [reply]
great, and thanks for reporting back! I've updated the docs now: https://plausible.io/docs/excluding
